lib2/package.json

```diff
@@ -1,36 +1,45 @@
 {
-    "name": "lib2",
-    "version": "1.0.0",
-    "files": [
-        "dist"
-    ],
-    "main": "./dist/index.js",
-    "module": "./dist/index.js",
-    "types": "./dist/index.d.ts",
-    "typings": "./dist/index",
-    "type": "module",
-    "exports": {
-        ".": {
-            "import": "./dist/index.js",
-            "require": "./dist/index.cjs",
-            "types": "./dist/index.d.ts"
-        }
+  "name": "lib2",
+  "version": "1.0.0",
+  "files": [
+    "/component",
+    "/dist"
+  ],
+  "type": "module",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
     },
-    "scripts": {
-        "build": "npx tsup --clean --minify"
+    "./component": {
```

Comment:
allowing a deep/direct import of `component` is necessary if you take the strategy of only rendering your component on the client, not the server, in nextjs. If you want to instead just upgrade styled-components to the beta version, you dont need to do this at all.